h/views/activity.py

```diff
@@ -106,7 +106,9 @@ def search(self):
 
         result['opts'] = {'search_groupname': self.group.name}
 
-        if self.request.user not in self.group.members:
+        # if the group has a concept of members (aka joinable_by is not None)
+        # and the user is not in the list of members
```

Comment:
On a small stylistic note, the general convention in our codebase is to use full sentences for docstrings and comments (`# If the group ... list of members.`)